zerver/management/commands/import.py

```diff
@@ -55,7 +59,12 @@ def handle(self, *args: Any, **options: Any) -> None:
         elif options["import_into_nonempty"]:
             print("NOTE: The argument 'import_into_nonempty' is now the default behavior.")
 
-        check_subdomain_available(subdomain, from_management_command=True)
+        allow_reserved_subdomain = False
+
+        if options["allow_reserved_subdomain"]:
+            allow_reserved_subdomain = True
+
+        check_subdomain_available(subdomain, allow_reserved_subdomain)
```

Comment:
Following up on my comment on the issue, specifically I think this should be:
```
try:
    check_subdomain_available(subdomain, allow_reserved_subdomain)
except ReservedSubdomainError:
    raise CommandError("Subdomain reserved; pass --allow-reserved-subdomain to use.")
```

Reply:
Ok Thanks! I would make the change